src/components/Download.tsx

```diff
@@ -38,54 +40,96 @@ function Download() {
             Description: column.description || '',
           };
 
-          if (column.standardizedVariable) {
-            dictionaryEntry.Annotations = {};
+          // Description of levels always included for the BIDS section
+          if (column.dataType === 'Categorical' && column.levels) {
+            dictionaryEntry.Levels = Object.entries(column.levels).reduce(
+              (levelsObj, [levelKey, levelValue]) => ({
+                ...levelsObj,
+                [levelKey]: {
+                  Description: levelValue.description || '',
+                },
+              }),
+              {} as { [key: string]: { Description: string } }
+            );
+          }
 
-            dictionaryEntry.Annotations.IsAbout = {
-              Label: column.standardizedVariable.label,
-              TermURL: column.standardizedVariable.identifier,
+          if (column.dataType === 'Continuous' && column.units !== undefined) {
+            dictionaryEntry.Units = column.units;
+          }
+
+          if (column.standardizedVariable) {
+            dictionaryEntry.Annotations = {
+              IsAbout: {
+                TermURL: column.standardizedVariable.identifier,
+                Label: column.standardizedVariable.label,
+              },
             };
 
+            // Handle level terms only for standardized columns
```

Comment:
This comment is not clear enough - and in fact a bit confusing. This sounds as if you **only** create *BIDS Level* descriptions if the column is mapped to a standardized variable. I don't think that's what you mean.

Reply:
The comment is meant to note that we only add/handle `Levels terms` if they're mapped to a standardized variable whether at the BIDS section or the Annotations field.